content-hour-row.php

```diff
@@ -66,136 +66,133 @@
 											<b>Today</b><br/>
 										<?php
 											$curDay = $now;
-											
-											$temp = $post;									
-											
-											//echo "S: ".date("Y-m-d", $today)." ";
-											
-											$message = getMessageDay($locationId, $today);
-											
-											if ($message != "" ) {
-												
-												if ($showMobileSpecial == 1) {
+
+											$temp = $post;
+
+											// echo "S: ".date("Y-m-d", $today)." "; .
```

Comment:
Unless there is some need for this extra code to be here, I would say we should delete it and rely on github history if we need to retrieve it. Then we won't get red x's. What do you think of that?